generators/client/index.js

```diff
@@ -315,6 +315,7 @@ module.exports = JhipsterClientGenerator.extend({
         writeSwaggerFiles: function () {
             // Swagger-ui for Jhipster
             this.template(MAIN_SRC_DIR + 'swagger-ui/_index.html', MAIN_SRC_DIR + 'swagger-ui/index.html', this, {});
+            this.template(MAIN_SRC_DIR + 'swagger-ui/_index-prod.html', MAIN_SRC_DIR + 'swagger-ui/index-prod.html', this, {});
```

Comment:
The differences are in the javascript import not passing by bower_components. I agree that this could be improved maybe by refactoring the common js code.